### ISSUE TITLE
Replacing FossAV

### ISSUE THREAD

**Issue by PPPDUD** [FossAV/FossAV#23]
FossAV has not had any recent developments, and it doesn't seem to be going anywhere, so I've taken the task of making a better FossAV from scratch. Anybody else interested?

**Comment by HydraDragonAntivirus** [comment]
you can look my project.

**Comment by Piggo4676** [comment]
Sure yeh this project sorta died and I'm not good at managing and stuff so sure

**Comment by UniqueName12345** [comment]
Yeah I don't really mind. I completely forgot about FossAV until I checked the following tab to see what Synkstar is up to (please don't ask why I'm following them).

**Comment by PPPDUD** [comment]
I am starting a project to reform FossAV without replacing it altogether (#24), can someone add me as a member?
